unittests/InstructionCountCI/PrimaryGroup.json

```diff
@@ -3375,43 +3287,36 @@
         "cmp x20, x21",
         "cset x24, lo",
         "orr x23, x23, x24, lsl #29",
-        "eor x21, x20, x21",
-        "eor x20, x22, x20",
-        "and x20, x21, x20",
-        "ubfx x20, x20, #15, #1",
+        "eor w21, w20, w21",
+        "eor w20, w22, w20",
+        "and w20, w21, w20",
+        "ubfx w20, w20, #15, #1",
         "orr x20, x23, x20, lsl #28",
         "str w20, [x28, #728]"
       ]
     },
     "neg ebx": {
-      "ExpectedInstructionCount": 20,
+      "ExpectedInstructionCount": 13,
       "Optimal": "No",
       "Comment": "GROUP2 0xf7 /2",
       "ExpectedArm64ASM": [
         "mov w20, #0x0",
         "lsr w21, w7, #0",
         "neg w7, w21",
-        "eor x22, x20, x21",
-        "eor x22, x22, x7",
-        "ubfx x22, x22, #4, #1",
+        "eor w22, w20, w21",
+        "eor w22, w22, w7",
+        "ubfx w22, w22, #4, #1",
         "strb w22, [x28, #708]",
         "eor x22, x7, #0x1",
         "strb w22, [x28, #706]",
-        "tst w7, w7",
-        "mrs x22, nzcv",
-        "cmp x20, x21",
-        "cset x23, lo",
-        "orr x22, x22, x23, lsl #29",
-        "eor x21, x20, x21",
-        "eor x20, x7, x20",
-        "and x20, x21, x20",
-        "ubfx x20, x20, #31, #1",
-        "orr x20, x22, x20, lsl #28",
+        "cmp w20, w21",
```

Comment:
Note to self: extend const prop to handle this so we translate to a `NEGS` instruction instead